working/augmentation-libraries/feature-specification.md

```diff
@@ -142,59 +141,58 @@ It is a compile-time error if:
     is an *augmenting* declaration, described below). *This is the same error
     conceptually as having a name collision in one file.*
 
-*   An augmentation library contains any `part` directives.
+*   An augmentation contains any `part` directives.
 
-### Applying an augmentation library
+### Applying an augmentation
 
-A main library applies an augmentation to itself using a new import directive
-that looks like this:
+A main library applies an augmentation to itself using a new `augment` directive
+which looks like this:
 
 ```dart
-import augment 'some_augmentation.dart';
+augment 'some_augmentation.dart';
 ```
 
 This directive tells the compiler to read the given library augmentation and
 merge its declarations into this library. It is a compile-time error if:
 
-*   The library referenced in an `import augment` directive is not an
-    augmentation and does not have a `library augment` directive.
+*   The URI referenced in an `augment` directive is not an augmentation and does
+    not have a `augment library` directive.
 
-*   The library referenced in a `library augment` directive does not have an
-    `import augment` directive pointing back to this augmentation.
+*   The library referenced in a `augment library` directive does not have an
+    `augment` directive pointing back to this augmentation.
 
-*   The same augmentation library is applied more than once. *In other words,
-    you can't have redundant `import augment` directives that point to the same
-    library.*
+*   The same augmentation is applied more than once. *In other words, you can't
+    have redundant `augment` directives that point to the same library.*
 
 *   The main library and its augmentations do not all have the same language
     version. There is only one user-visible library at the end, and it should
     have a consistent version across its entire surface area. *An augmentation
-    library does not automatically inherit any language version from the main
-    library and may need an explicit language version comment of its own in
-    order to adhere to this requirement.*
+    does not automatically inherit any language version from the main library
+    and may need an explicit language version comment of its own in order to
+    adhere to this requirement.*
 
-Since the main library and its augmentation both point to each other, these
+Since the main library and its augmentations both point to each other, these
 rules imply that a given augmentation file can only be used to augment a single
 library.
 
 ### Merge order
 
 A library may apply multiple augmentations to itself. Also, augmentation files
-may themselves contain `import augment` directives. The entire tree of
-augmentations is recursively applied to the main library. The merge order is
-defined as a depth-first pre-order traversal of the `import augment` directives
-starting at the main library.
+may themselves contain `augment` directives. The entire tree of augmentations is
```

Comment:
"`augment` directives" have something of the same issue as plain "augmentation", because I'm pretty certainly going to use *something like that* that to cover `augment class` declarations too, which an augmentation file can also contain.
If the distinction is that the "augment import" is a *directive* and the other ones are *declarations*, I don't think that's going to be distinct enough. 
At least not unless we make that distinction *very explicit* where we introduce the directive above.

Say, something like:
> A directive of the form `augment "file.dart";` which applies a library augmentation to the library of the directive, is called an _`augment` directive_, which distinguishes it from _`augment` declarations_, which are declarations that an augmentation file can also contain, and whose syntax also starts with `augment`. These `augment` declarations will be introduced later.

If we do that, very up front, I think we *can* use "`augment` directive" unambiguously.
(It's really about defining the term. If we do nothing, "`augment` directive" is just a descriptive term, not a declared concept, and people can interpret it, and interpret it incorrectly. If we *define* the meaning, then the term has only that one meaning in this document. I'm a big fan of giving things defined names.)